k8s/aat/common/sscs/sscs-tya-notif.yml

```diff
@@ -0,0 +1,27 @@
+---
```

Comment:
this must be missing in codeowners btw...
SSCS should be getting review requests...

Reply:
It is in there @timja: https://github.com/hmcts/cnp-flux-config/blob/tribunals-flux-config/CODEOWNERS#L112. Maybe because that is yml and not yaml as stated in code owners